features/step_definitions/money_manager_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
```

Comment:
@Tr4pSt3R Since the steps definition is very specific to the Money Manager tool, we named it `money_manager_steps` and named the feature `clear_data_on_signout` because we felt it better explained what is going on. If we add more sign out enhancements we can rename it to `enhance_sign_out_behaviour`.